lib/ruumba/parser.rb

```diff
@@ -0,0 +1,73 @@
+# frozen_string_literal: true
+
+# @author Eric Weinstein <eric.q.weinstein@gmail.com>
+
+module Ruumba
+  # Responsible for extracting interpolated Ruby.
+  class Parser
+    # The regular expression to capture interpolated Ruby.
+    ERB_REGEX = /<%[-=]?(.*?)-?%>/m
+
+    # Extracts Ruby code from an ERB template.
+    # @return [String] The extracted ruby code
+    def extract(contents)
+      file_text, matches = parse(contents)
+
+      extracted_ruby = +''
```

Comment:
Why the `+` here?

Reply:
I had added `# frozen_string_literal: true` to the file, so I needed to use `+''` to initialise an unfrozen string.